packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -188,10 +189,12 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
 
         this.logger = ChildLogger.create(
             dataStoreContext.logger,
-            "FluidDataStoreRuntime",
+            undefined,
             {all:{ dataStoreId: uuid() }},
         );
 
+        this.runtimeLogger = ChildLogger.create(this.logger, "FluidDataStoreRuntime");
```

Comment:
Same as above - I think we should standardize on a naming convention of `this.logger` for the namespaced thing to use within the class, and `this.baseLogger` for the parent of `this.logger`, which often has domain-specific props added but not this namespace.

@anthony-murphy had a good suggestion (I was skeptical at the time but thought through my objections and on board now) to actually keep track of a ChildLogger's given baseLogger, and expose it on there, so classes like this don't have to have two logger references.  When creating a nested thingy just pass `this.logger.baseLogger`, and never call `this.logger.baseLogger.sendFoo`.  Could be a later PR, this one as-is is great, if we align on names.